### ISSUE TITLE
discussion: using vertx-web router

### ISSUE THREAD

**Issue by gjesse** [cy6erGn0m/vertx3-lang-kotlin#3]
Hi, I started playing around with this code today, but quickly hit a limitation in that the router provided doesn't support path parameters. with some digging & poking, I ended up with this: https://github.com/gjesse/vertx-kotlin-router, which provides a similar interface but delegates to the standard vertx router. I'm wondering if you'd be amenable to swapping out the existing router implementation with something like what I've got? If so i can work up a proper PR with tests,etc.

**Comment by cy6erGn0m** [comment]
yes, I was thinking about extending bindings to support vertx-web as well. You are welcome to make PR ;)

**Comment by gjesse** [comment]
do you want to keep the barebones router around too, or scrap it?

**Comment by cy6erGn0m** [comment]
@gjesse you could use different package for now but you don't need to be compatible with old router

**Comment by gjesse** [comment]
Ok. Some things have changed & so I don't know when i might be able to get around to this now, but i will attempt to if I'm able.
